app/src/main/res/values-nl/strings.xml

```diff
@@ -17,7 +17,7 @@
   -->
 
 <resources>
-    <string name="app_copyright">"Copyright \u00A9 2015 \u002D "</string>
```

Comment:
While "Auteursrecht" isn't incorrect, the word "Copyright" is more common in digital projects in The Netherlands.

Reply:
Copyright doesn't exist in Dutch law. Auteursrecht does. Lets not stick to common errors just because they are common. Auteursrecht is a well recognized word as well and is correct.

Reply:
As far as I can discover both are correct to use on a website or app but they do not have any additional meaning to the law.

Sources (in Dutch):
https://www.pumbo.nl/faq/auteursrecht/verschil-tussen-auteursrecht-copyright
https://nl.wikipedia.org/wiki/Auteursrecht
https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/copyright
https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/auteursrecht

I do prefer the common "Copyright" but "Auteursrecht" wouldn't be a big dealbreaker for me.  I  just wanted to start a discussion which of both should be used. It would be great if more Dutch people visit this respository and give their opinion.